core/che-core-api-core/src/main/java/org/eclipse/che/api/core/ErrorCodes.java

```diff
@@ -1,38 +0,0 @@
-/*
- * Copyright (c) 2012-2018 Red Hat, Inc.
- * This program and the accompanying materials are made
- * available under the terms of the Eclipse Public License 2.0
- * which is available at https://www.eclipse.org/legal/epl-2.0/
- *
- * SPDX-License-Identifier: EPL-2.0
- *
- * Contributors:
- *   Red Hat, Inc. - initial API and implementation
- */
-package org.eclipse.che.api.core;
-
-/**
- * Error codes that are used in exceptions. Defined error codes MUST BE in range <b>15000-32999</b>
- * inclusive.
- *
- * @author Igor Vinokur
- */
-public class ErrorCodes {
-  private ErrorCodes() {}
```

Comment:
what is the reason to move it to a different location?

Reply:
It is removed as unused anymore